samples/excel/89-preview-apis/chart-trendlines.yaml

```diff
@@ -0,0 +1,203 @@
+order: 1
+id: excel-89-preview-apis-chart-trendlines-
+name: Chart trendlines
+description: 'Add, get, set and format trendlines to a chart.'
+host: EXCEL
+api_set:
+    ExcelAPI: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-trendline-polynomial").click(() => tryCatch(addTrendlinePolynomial));
+        $("#get-trendline").click(() => tryCatch(getTrendlineType));
+        $("#get-trendline-color").click(() => tryCatch(getTrendlineColor));
+        $("#set-trendline-color").click(() => tryCatch(setTrendlineColor));
+
+        async function addTrendlinePolynomial() {
```

Comment:
Suggest that you add logic to this method such that a user can only add one trendline to the chart. i.e., if they click the "Add trendline" button more than once, it should show an alert that says "You have already added a trendline to this chart. Choose the other buttons to get trendline type, get trendline color, and set trendline color." (or something like that).

Currently if you click the "Add trendline" button multiple times, it adds a new trendline to the chart each time,....but the other functions (get trendline type, get color, set color) always just operate on the first trendline.